.azure/gpu-tests-fabric.yml

```diff
@@ -101,6 +101,7 @@ jobs:
     - bash: |
         extra=$(python -c "print({'lightning': 'fabric-'}.get('$(PACKAGE_NAME)', ''))")
         pip install -e ".[${extra}dev]" pytest-timeout -U --find-links ${TORCH_URL}
+        pip install -U deepspeed
```

Comment:
TODO: revert

Reply:
waht was this added at the first place? You say that `deepspeed` was not installed event though it was required in startegies.txt, which is part of dev extra...

Reply:
This was to force the CI to upgrade deepseed to latest. Even if the version is specified in requirements.txt, it won't install a new version of an existing one is already installed. This was necessary to ensure the changes in this branch work with the latest version.